README.md

```diff
@@ -108,9 +108,9 @@ async def main():
       ),
       on_clicked=lambda: print("Notification clicked"),
       on_dismissed=lambda: print("Notification dismissed"),
-      sound=True,
+      sound_file=DEFAULT_SOUND,
   )
-  
+
```

Comment:
@samschott you might want to make sure you have "delete trailing whitespace" turned on in your favorite code editor to avoid diffs like this in the future.

Reply:
Fair point. It's been seem time ago since I wrote those docs.